translator/trace/jaegerthrift_to_protospan_test.go

```diff
@@ -208,6 +208,20 @@ func TestConservativeConversions(t *testing.T) {
 						Code:    5,
 						Message: "cache miss",
 					},
+					Attributes: &tracepb.Span_Attributes{
+						AttributeMap: map[string]*tracepb.AttributeValue{
```

Comment:
We want to capture the status code and not capture it as an attribute otherwise there is no point in setting the status above. What's your rationale for this change?